src/styles/components/_sage-form.scss

```diff
@@ -31,14 +28,13 @@
     span {
       white-space: pre-line;
       display: flex;
-      margin-bottom: 5px;
+      font-size: 1.6rem;
```

Comment:
does this need to be responsive ? is it ok to keep 16px for both mobile and desktop ?